app/components/Map.js

```diff
@@ -1,24 +1,94 @@
-import React, { Component, PropTypes, StyleSheet, Text, View  } from 'react-native';
+
+import React, { Component, PropTypes, StyleSheet, Text, View, Dimensions, AlertIOS  } from 'react-native';
+
+import MapView from 'react-native-maps';
+
 
 export default class Map extends Component {
   constructor(props) {
     super(props);
     this.state = {
+      position: {
+        latitude: 37.78825,
+        longitude: -122.4324,
+        latitudeDelta: 0.0922,
+        longitudeDelta: 0.0421,
+      },
+      lastPosition: undefined,
+      pins: [],
+      walker: undefined
     };
+
+    // this.currentPosition = new Firebase("https://interruptedlobster.firebaseio.com/testPosition");
   }
+
+  // listenForItems(itemsRef) {
+  //     itemsRef.on('value', (snap) => {
+  //       // get children as an array
+        
+  //       this.setState({
+  //         firebasePosition: snap
+  //       });
+  //     });
+  //   }
+
+  // componentDidMount() {
+  //   navigator.geolocation.getCurrentPosition(
+  //     (position) => {
+  //       var initialPosition = JSON.stringify(position);
+  //       this.setState({walker: initialPosition});
+  //     },
+  //     (error) => {
+  //       alert(error.message)
+  //     },
+  //     {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000}
+  //   );
+
+  //   this.watchID = navigator.geolocation.watchPosition(
+  //     (position) => {
+  //       var lastPosition = JSON.stringify(position);
+  //       // update firebase with current position
+  //       // this.currentPosition.set(position);
+  //       this.setState({lastPosition: position});
+  //     }
+  //   );
+  // }
```

Comment:
Let's remove this zombie code, we can always look at previous commits to see the previous code.